### PR TITLE
Changelog for 2.1.0

### PR DESCRIPTION
**Pull request by nparsons08**
connected to waffleio/waffle.io-takeout#100

### CODE REVIEW

File: releases/2.0.3.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+## Waffle Takeout 2.0.3
+_Released 2016.01.19_
```
Comment (homeyer) [inline releases/2.0.3.md]: let's update the date before releasing

**Comment by homeyer** [comment]
:construction: waiting until we cut a build to merge the changelog, and we'll incrementally add to it until then

**Comment by ashumz** [comment]
update README.md with links to the new releases

**Comment by marydavis** [comment]
:+1:

**Pull request merged**